source/basic.d.ts

```diff
@@ -32,7 +32,7 @@ export type JsonPrimitive = string | number | boolean | null;
 /**
 Matches any valid JSON value.
 
-@remarks Also see ./Jsonify
+@see Also see ./Jsonify
```

Comment:
This is not correct usage of `@see`. I specifically asked you to make sure you used it correctly.

Reply:
I added 8bf303296bcbd0bfb8291d0e219c0602d017f5f8

This is similar to https://github.com/sindresorhus/type-fest/blob/6b8bee7d0df2bfce9cab4ae59bb6b14f81d7a633/source/entries.d.ts#L13, but different than others like https://github.com/sindresorhus/type-fest/blob/6b8bee7d0df2bfce9cab4ae59bb6b14f81d7a633/ts41/delimiter-case.d.ts#L25